kaldb/src/test/java/com/slack/kaldb/metadata/zookeeper/ZooKeeperCachedMetadataStoreImplTest.java

```diff
@@ -31,18 +32,26 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class CachedMetadataStoreImplTest {
-
-  private static final Logger LOG = LoggerFactory.getLogger(CachedMetadataStoreImplTest.class);
+public class ZooKeeperCachedMetadataStoreImplTest {
+  private static final Logger LOG =
+      LoggerFactory.getLogger(ZooKeeperCachedMetadataStoreImplTest.class);
+  private static final SnapshotMetadata defaultRoot = makeSnapshot("testRootData");
+  private final String defaultRootStr;
 
   private TestingServer testingServer;
   private ZookeeperMetadataStoreImpl metadataStore;
   private MeterRegistry meterRegistry;
   private final SnapshotMetadataSerializer serDe = new SnapshotMetadataSerializer();
   private ZooKeeper zooKeeper;
+  private int expectedCacheErrorCount = 0;
+
+  public ZooKeeperCachedMetadataStoreImplTest() throws InvalidProtocolBufferException {
+    defaultRootStr = serDe.toJsonStr(defaultRoot);
+  }
 
   static class CountingCachedMetadataListener implements CachedMetadataStoreListener {
     private int cacheChangedCounter = 0;
+    // TODO: Add a test with state changed counter.
```

Comment:
Is there a reason we're not including this test now?

Reply:
I added the TODO assuming we were missing the test. After digging in I realized that this method is not called in current implementation.I have a diff to update the interface, but then realized having this interface now is cheaper than adding it later. So leaving it in for now. If you feel strongly, I can remove it for now.